apps/docs/src/guide/types/arrays.md

```diff
@@ -1,20 +1,35 @@
 # Arrays
 
-## Passing in arrays
+In Sway, an `Array` is a fixed-size collection of elements of the same type, similar to a `Tuple`. `Arrays` can hold arbitrary types, including non-primitive types, with their size determined at compile time.
 
-You can pass a JavaScript `Array` into your contract method transparently, allowing passthrough of expected data via the type regardless of Array type. An Array in Sway is fixed in size, so maximum storage size is determined at time of program compile, not during runtime.
+## Using Arrays in the SDK
 
-An Array is just a Typed Array:
+You can pass a TypeScript `Array` into your contract method seamlessly, allowing the expected data to be passed through the type regardless of the `Array` type. In Sway, `Arrays` are fixed in size, so the storage size is determined at the time of program compilation, not during runtime.
```

Comment:
```suggestion
You can pass a TypeScript `Array` into your contract method seamlessly just like you would pass an `Array` to a TypeScript function. 

The SDK handles the conversion from TypeScript to Sway in the background, allowing the expected data to be passed through the type regardless of the `Array` type. 

// Let's move this piece of text to below the code snippet maybe?
In Sway, `Arrays` are fixed in size, so the storage size is determined at the time of program compilation, not during runtime.
```

This piece of text could come off as a little unclear and cluttered. Thoughts on this rewording?